src/strategies/infinityprotocol-liquidity-pools/README.md

```diff
@@ -0,0 +1,27 @@
+# Liquidity Providers
+
+This strategy will return the scores of all users who have provided token liquidity on any Uniswap style exchange. Users can change the subGraphURL field to direct their request to a different subgraph. 
+
+
+## Example
+
+The space config will look like this:
+
+```JSON
+{
+  "strategies": [
+    ["infinityprotocol-liquidity-pools", {
+      // token parameters
+    "params": {
+        "address": "0xc168e40227e4ebd8c1cae80f7a55a4f0e6d66c97",
+        "symbol": "DFYN",
+        "decimal": 18,
```

Comment:
the decimal option is never used in the strategy file
```suggestion
        "symbol": "DFYN",
```

Reply:
You're right. Removed the decimal field.